BCStudentSoftwareDevTeam/cas
# clicking edit course x modal takes you to the right pages

Original issue is that clicking on the x button in the modal of editCourse.html always took the user to the courses page. However, it is reused in course management, so it should redirect to course management on that page. I fixed the issue by copying the code from the footer close button, which does what the x button was supposed to do. 

To Test:
Create a course in the courses page and make it crosslisted (so it shows up in course management). Clicking edit and hitting the x should keep you on the courses page. 
Go to the course management page. Clicking edit and hitting the x should keep you on the course management page.
